amundsen_application/api/utils/request_utils.py

```diff
@@ -6,7 +6,7 @@
 REQUEST_SESSION_TIMEOUT_SEC = 3
 
 
-def get_query_param(args: Dict, param: str, error_msg: str = None) -> str:
+def get_query_param(args: Dict, param: str, error_msg: str = None):  # type: ignore
```

Comment:
MyPy was giving a few errors as get_query_params was returning non-str objects that were valid

Reply:
Ah just noticed your comment on this. I'm not a fan of loosening the typing on existing code. Perhaps the mistake was recommending that you use this method instead of creating a new one for your purposes given that your use case is different. 

Another option instead of using `get_query_param` could just be the simpler approach of using `.get()` and passing in some safe defaults. But how about I pose this instead -- what do you think will be best to do in the case where the React app has some issue and doesn't pass the right arguments? Let's agree on the behavior before I make any other recommendation.

Reply:
I feel like if any data's malformed or it doesn't exist then an exception should be thrown and a 500 should be returned

Reply:
@ttannis if that's the case, then directly accessing the dict would be fine as a KeyError would be thrown if the data is in fact malformed.

Reply:
I see let's go with that then. Apologies for the bit of extra churn.